torch/_dynamo/variables/torch.py

```diff
@@ -269,6 +270,15 @@ def call_function(
             return ConstantVariable(torch.is_grad_enabled(), **options).add_guards(
                 GradModeVariable._guards_singleton
             )
+        elif self.value is torch.cuda.stream:
+            log.warning(
+                "torch.cuda.stream can't be passed through AOT yet, will be ignored"
+            )
+            return CUDAStreamContextVariable.create(
+                tx, args[0].as_python_constant(), **options
+            )
+        elif self.value is torch.cuda.streams.Stream:
+            return TorchVariable(self.value(), **options)
```

Comment:
This doesn't look right.  It is calling stream constructor at compile time rather than runtime.